ui/app/models/pki/tidy.js

```diff
@@ -49,6 +49,24 @@ export default class PkiTidyModel extends Model {
   })
   intervalDuration; // auto-tidy only
 
+  @attr('string', {
```

Comment:
Definitely a good thing for the label field. 

As for the default, since the backend always returns a response with the field values set, should the UI specify something? It seemed like it would always pull the value from the auto-tidy config read endpoint?

Reply:
Yes - setting a default doesn't affect how the tidy config saves (or the read view where the config details display). The difference is purely UX. However, I'm remembering now that we did setup this form to first request any existing tidy configs, which should include defaults, and pre-fill the form with those. (Usually forms create a new record, so there is know awareness of defaults until _after_ saving) 

<img width="1086" alt="Screenshot 2024-09-25 at 1 13 01 PM" src="https://github.com/user-attachments/assets/e5388a26-7ab2-49d7-abca-f837b240336b">

`interval_duration` works as expected and 12 hours is pre-filled in the input. But I noticed `safety_buffer` below is empty. This is because we've typed it as a 'string' in the model, but the value is returned from the read request as an integer. 

Another option is mentioning the default in the subtext. (Which we do for safety buffer) Since we're not consulting design here, it can be up to us. I prefer defaults existing in the form since it's more transparent to the user what is being set as a default (often subtext is overlooked). 

But, you're right that in this case there should be no need to explicitly set it in the form since we make the read request first. Though, I recommend removing the type `'string'` so it populates the input correctly. Setting the type on the model is an ember data thing that provides some default serialization "help" that is actually unhelpful in this instance. Often for ttls we don't define a type because they can be strings or integers (notice how `intervalDuration` in the model doesn't have a type defined) While you're in this file, it may be helpful to also remove the 'string' type for `safetyBuffer`

Sorry for the long-winded reply! 😂